client/js/libs/handlebars/parse.js

```diff
@@ -14,7 +14,7 @@ function uri(text) {
 			return url;
 		}
 		var split = url.split("<");
-		url = "<a href='" + split[0].replace(/^www/, "//www") + "' target='_blank'>" + split[0] + "</a>";
+		url = "<a href='" + split[0].replace(/^www/, "//www") + "' target='_blank' rel='noopener'>" + split[0] + "</a>";
```

Comment:
https://mathiasbynens.github.io/rel-noopener/

Thought it would be a good idea to include along with `referrer no-referrer`